### PR TITLE
Cairo: make status() public

### PR DESCRIPTION
**Pull request by pbor**
This starts to address some of the less ambiguous parts of https://github.com/gtk-rs/gtk-rs/issues/74

### CODE REVIEW

File: cairo/src/context.rs [hunk 1]
```diff
@@ -815,9 +815,9 @@ mod tests {
     }
 
     #[test]
-    fn clip_rectange() {
+    fn clip_rectangle() {
```
Comment (GuillaumeGomez) [inline cairo/src/context.rs:818]: Nice typo fix! :)

File: cairo/src/context.rs [hunk 2]
```diff
@@ -815,9 +815,9 @@ mod tests {
...
         let ctx = create_ctx();
-        let rect = ctx.copy_clip_rectangle_list();
+        let rect = ctx.copy_clip_rectangle_list()?;
```
Comment (GuillaumeGomez) [inline cairo/src/context.rs:820]: Since the function doesn't return a `Result`, I think it'll fail. Better use a `expect` here I think.

**Review by GuillaumeGomez — APPROVED** [review]

**Comment by GuillaumeGomez** [comment]
Apart from the fmt failure, looks good to me!

File: cairo/src/context.rs (hunk 2 above)
Comment (pbor) [reply cairo/src/context.rs]: Indeed that did not even compile, I forgot to amend

**Comment by pbor** [comment]
> Apart from the fmt failure, looks good to me!

Fixed. thanks for the quick review!

**Review by bilelmoussaoui — APPROVED** [review]

**Comment by GuillaumeGomez** [comment]
Please remove the update on the `gir` submodule too. :)

File: cairo/src/context.rs (hunk 2 above)
Comment (pbor) [reply cairo/src/context.rs]: > Please remove the update on the `gir` submodule too. :)

I wonder how many silly mistakes I can still make in a row :(
Hopefully ok now

**Pull request merged**